crocket/src/main/java/com/crocket/Ball.java

```diff
@@ -20,13 +20,24 @@ public double getyVelocity(){
         return yVelocity;
     }
 
+    public void setxVelocity(double xVelocity){
+        this.xVelocity = xVelocity;
+    }
+
+    public void setyVelocity(double yVelocity){
+        this.yVelocity = yVelocity;
+    }
+    
     public void startBall(double cosinus, double sinus, double power){
         this.xVelocity = (power*cosinus)/mass;
         this.yVelocity = (power*sinus)/mass;
     }
-
+    
     public void move(){
         this.setxPosition((this.getxPosition()+xVelocity));
         this.setyPosition((this.getyPosition()+yVelocity));
+        
```

Comment:
Don't understand why you use setPosition in move but not setVelocity in startBall

Reply:
Not sure. I don't think I implemented this code